src/components/Header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import hamburgerMenu from "@/assets/images/icon-menu.svg";
+import logoSvg from "@/assets/images/logo.svg";
+import cartSvg from "@/assets/images/icon-cart.svg";
+import avatarPng from "@/assets/images/image-avatar.png";
+import styles from "./Header.module.sass";
+import Image from "next/image";
+
+export default function Header({
+  toggleSidebar,
+}: {
+  toggleSidebar: () => void;
+}) {
+  return (
+    <header className={styles.header}>
+      <section className={styles.headerLeftSide}>
+        <button onClick={toggleSidebar} className={styles.menuButton}>
+          <Image src={hamburgerMenu} alt="hamburger menu" />
+        </button>
+      </section>
+      <section className={styles.headerRightSide}>
+        <div className="logo">
+          <Image
+            className={styles.companyLogo}
+            src={logoSvg}
+            alt="sneakers company logo"
+          />
+        </div>
+        <div className={styles.userAndAvatar}>
+          <button className={styles.cartButton}>
```

Comment:
Please add aria-label if the children only contain icons. same with the others

Reply:
Added aria-labels for containers and also buttons